<launch>
  
    <arg name="namespace" default="robot_1" />
    
    <group ns="$(arg namespace)/trajectory">
        <!-- Launch the nodes to publish all the robot tfs and visualize in rviz -->
        <node name="robot_state_publisher" pkg="robot_state_publisher" type="robot_state_publisher">
            <remap from="joint_states" to="visualization/joint_states/trajectory"/>
            <param name="tf_prefix" value="trajectory"/>
        </node>
    </group>

    <group ns="$(arg namespace)/ground_truth">
        <!-- Launch the nodes to publish all the robot tfs and visualize in rviz -->
        <node name="robot_state_publisher" pkg="robot_state_publisher" type="robot_state_publisher">
            <remap from="joint_states" to="visualization/joint_states/ground_truth"/>
            <param name="tf_prefix" value="ground_truth"/>
        </node>
    </group>

    <!-- Launch the node to publish the visual topics for the plans -->
    <node name="rviz_interface" pkg="quad_utils" type="rviz_interface_node" output="screen"/>

</launch>
